app/src/command.rs

```diff
@@ -32,6 +32,15 @@ use {
 
 impl<D: Dependencies> Application<D> {
     pub fn start_block_build(&mut self, attributes: Payload, id: PayloadId) {
+        if self
+            .payload_queries
+            .by_id(&self.storage_reader, id)
+            .unwrap()
```

Comment:
Since this is just an early exit, we could avoid the `unwrap` here with `.ok().flatten()`. Although there are other unwraps in the function from storage access, so probably this function should just be refactored to return a `Result`. It can be done in another PR, probably as part of the work for #247